README.md

```diff
@@ -0,0 +1,31 @@
+# Awesome Books
+
+> Books list web application.
+
+This project is a basic website that allows users to add books to a list. The books list is store locally. So you can close your browser and when you open it again you will still have you books list.
+
+## Built With
+
+- HTML
+- CSS
+- JS
+
+# Live Version
+
+https://mrcbq.github.io/awesome-books/
+
+##👥 Authors
+
+ **Marco Antonio Baquero**
+
+- GitHub: [@Marco Antonio](https://github.com/mrcbq)
+
+ **Juan Diego Guerra**
+
```

Comment:
- [ ] Kindly Use the [new Microverse README template](https://github.com/microverseinc/readme-template) and edit its sections with your project and personal data.